### ISSUE TITLE
Choose ML/DL framework

### ISSUE THREAD

**Issue by ManuelJNunez** [ManuelJNunez/TFG#14]

**Comment by ManuelJNunez** [comment]
Primero voy a usar `PyTorch` porque es fácil de aprnder y muy potente.

**Comment by ManuelJNunez** [comment]
Y también `scikit-learn` porque trae funciones para calcular la accuracy y el `F1 score` entre otras muchas utilidades para el Machine Learning.
